--- drivers/gpu/arm/bifrost/backend/gpu/mali_kbase_devfreq.h.old	2023-05-08 00:06:57.575734326 +0200
+++ drivers/gpu/arm/bifrost/backend/gpu/mali_kbase_devfreq.h	2023-05-08 00:06:34.089468258 +0200
@@ -26,7 +26,8 @@
 
 void kbase_devfreq_term(struct kbase_device *kbdev);
 
-int kbase_devfreq_opp_helper(struct dev_pm_set_opp_data *data);
+struct dev_pm_set_opp_data *data;
+int kbase_devfreq_opp_helper(struct dev_pm_set_opp_data bastien);
 
 
 /**
